lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1052,6 +1052,9 @@ def _create_deployment_parameters(
         ):
             arm_parameters.use_availability_sets = True
 
+        if self._azure_runbook.enable_hibernation:
```

Comment:
With the new implementation, the arm parameter is not necessary now. The on_before_deployment can update the arm template without any parameters.